src/lib/pages/frontsite/Documents/DocumentDetails/DocumentMetadata/DocumentMetadataTabs/DocumentMetadataTabs.js

```diff
@@ -120,6 +126,7 @@ class DocumentMetadataTabs extends Component {
         panes={this.renderTabPanes()}
         onTabChange={this.onTabChange}
         id="document-metadata-tabs"
+        className="document-metadata-tabs"
```

Comment:
Here we are using the same value for `id` which is meant to be unique among the dom elements, but also for class name which is meant to be shared among elements that belong to the same class. Personally I would go for something more explicit for the `id`.

Reply:
for me the `#document-metadata-tabs` and `.document-metadata-tabs` are very different, but I guess you mean for search in the code purposes. Fixed